nemo/lightning/pytorch/strategies.py

```diff
@@ -210,6 +210,12 @@ def connect(self, model: pl.LightningModule) -> None:
         if _maybe_mcore_config:
             self._mcore_config = _maybe_mcore_config
 
+        if hasattr(self._precision_plugin, 'dtype_config'):
```

Comment:
`grad_reduce_in_fp32` needs to go to the DDPConfig, doesn't seem like that will happen currently.

Reply:
Thank you @maanug-nv , very much appreciating you catching this!